index.html

```diff
@@ -4,33 +4,52 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio-Website</title>
+    <title>Bhushan's-Portfolio</title>
     <link rel="stylesheet" href="style.css" />
+   
   </head>
   <body>
     <header class="container">
       <a href="#" class="myLogo">
-        <img src="MyLogo.png" />
+        <img src="MyLogo.png" alt="logo"/>
       </a>
+         <!-- hamburger section -->
       <nav class="menuButtonMain">
-        <a href="#">
+        <a href="#" class="hamburgerOpen">
```

Comment:
- [ ] IT seems you have a white space at the right of your mobile screen, this is not in the design and makes your portfolio look a bit unprofessional, I'll suggest you adjust your CSS to fix this as it also affects your mobile menu. :+1: :100: 

- - SCREENSHOT:
![Screenshot from 2023-06-26 15-09-25](https://github.com/bhushan354/Portfolio_Project/assets/107632780/f709d78f-a3ce-45f0-90bb-ccd972afa02f)


- [OPTIONAL] As you build your portfolio gradually, your portfolio is meant to be responsive and look good across different screens, Currently this section in the screenshot below tends to stack on top of each other on some screens, I'll suggest you look into this. :smile: :+1: 

- - SCREENSHOT:
![Screenshot from 2023-06-26 14-57-53](https://github.com/bhushan354/Portfolio_Project/assets/107632780/ebb10919-d420-4e8d-8755-7077953bcbc4)